src/commands/get.js

```diff
@@ -35,9 +40,13 @@ export const actionCreator = () => async ({
 		throw new ValidationError('Unsupported type.');
 	}
 
-	return query.handlers[deAlias(type)](input, { testnet }).then(
-		processQueryResult(type),
-	);
+	const overrideTestnet = checkBoolean(testnet)
```

Comment:
This sounds like a function name. Prefer `testnetOverrideValue` or something similar.